templates/component/name.component.js

```diff
@@ -5,6 +5,7 @@ angular.module('<%= scriptAppName %>')
     template: '<h1>Hello {{ $ctrl.message }}</h1>',
     bindings: { message: '<' },
     controller: function() {
-      this.message = 'World'
+      var vm = this;
+      vm.message = 'World';
```

Comment:
I'd just keep the `this.message = 'World';`. No need to create a new reference to `this.`